common/app/Panes/redux/index.js

```diff
@@ -164,7 +164,8 @@ export default function createPanesAspects({ createPanesMap }) {
           const leftPane =
             state.panesByName[getPaneName(state.panes, paneIndex - 1)] || {};
           const rightBound = (rightPane.dividerLeft || 100) - dividerBuffer;
-          const leftBound = (leftPane.dividerLeft || 0) + dividerBuffer;
+          const leftBound = leftPane.name === 'Map' ? dividerBuffer
```

Comment:
Not a fan of coupling the logic here to the content of the pane. 

The logic that finds the leftpane should be changed to iterate to the next leftPane if the current one is hidden.

Reply:
You're right. In hindsight that's not a very stable solution. How about this? Just tested and it works for all pane open/closed variations.
```
const leftBound = leftPane ? dividerBuffer : (leftPane.dividerLeft || 0) + dividerBuffer;
```

Reply:
leftPane is always defined so leftBound is always dividerBuffer.